vite.config.ts

```diff
@@ -10,6 +10,7 @@ export default defineConfig({
     vue(),
     vueDevTools(),
   ],
+  base: "/fantasy-roller/",
```

Comment:
How come this was needed?

Reply:
discussed and resolved

Reply:
To point to the repository GH Pages address